packages/models-library/src/models_library/projects_pipeline.py

```diff
@@ -29,29 +30,69 @@ class ComputationTask(BaseModel):
     pipeline_details: PipelineDetails = Field(
         ..., description="the details of the generated pipeline"
     )
+    iteration: Optional[PositiveInt] = Field(
```

Comment:
NOTE that there is already a concept for iteration in meta-modeling. From the context i think it is clear (here the resource is a ``ComputationTask``) but it would be good to add at least a note in the description.

Reply:
hmm mine is prior to meta... I should win this fight

Reply:
the description is currently: ```the iteration id of the computation task (none if no task ran yet)``` is that not clear enough?